src/components/Demo.svelte

```diff
@@ -196,6 +280,15 @@ function onToggleClick(e) {
   toggleTheme();
 }
 
+function onSelectionChanged(e) {
+  console.log(selection);
+  const language = reverseLanguageAliases[selection.language];
+  const query = `?language=${language}`;
+  const newUrl = `${window.location.protocol}//${window.location.host}${window.location.pathname}${query}`;
+  window.history.replaceState(null, "", newUrl);
+  navigator.clipboard.writeText(newUrl);
```

Comment:
Please avoid writing to the clipboard here.
I see how my intention was perhaps interpreted to mean this - sorry.

I just want the URL to change, no changing the clipboard.